markdowns/1.md

```diff
@@ -7,39 +7,28 @@ date: "2020-09-03"
 id: "tutorial1md"
 --->
 
-# Build Your First QA System
+# Build Your First Question Answering System
 
-<img style="float: right;" src="https://upload.wikimedia.org/wikipedia/en/d/d8/Game_of_Thrones_title_card.jpg">
+- **Level**: Beginner
+- **Time to complete**: 20 minutes
+- **Prerequisites**: Prepare the Colab environment. See links below.
```

Comment:
I don't see any instructions if I want to run this on my computer. I think this would be enough to solve this issue. I am not sure what extra should be added
```suggestion
- **Prerequisites**: Prepare the Colab environment or your local machine. See links below.
```

Reply:
Yes I was a little extra careful here because there are some complications once we suggest that this code can be used locally:
- Haystack installation can differ depending on OS and environment
- There are different Elasticsearch initialisation options available depending on whether Docker is available or not
- GPU set up is different in Colab, Linux and Mac (Apple Silicon)
- GPU is not be available on Mac (Intel) and Windows

I think that the Haystack code scripts that @masci is pushing for in the Haystack repo will be something we can really advertise as code that works in your env. But for these tutorials, I would lean towards being Colab specific.